client/public/index.html

```diff
@@ -5,6 +5,9 @@
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
     <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.0/mapbox-gl.css' rel='stylesheet' />
```

Comment:
upgrade this component to v1.0.0 - `href='https://api.tiles.mapbox.com/mapbox-gl-js/v1.0.0/mapbox-gl.css'`